### PR TITLE
fix: responsive fix

### PR DESCRIPTION
**Pull request by vrx29**
Fix : 
- Mobile view 
- Hamburger icon and github icon updated
- fully mobile supported app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,71 +5,94 @@
...
-                <img class="terminal" src="./assets/terminal.svg" alt="hero"/>
-                <img class="mover" src="./assets/mover.svg" alt="hero"/>
-                <!-- <img src="./assets/hero.svg" alt="hero"/> -->
-            </div>
+      <!-- Hero Right Section Start -->
+      <div class="hero-right">
+        <div class="hero-img-cont">
+          <img class="terminal" src="./assets/terminal.svg" alt="hero" />
+          <img class="mover" src="./assets/mover.svg" alt="hero" />
+          <!-- <img src="./assets/hero.svg" alt="hero"/> -->
```
Comment (ApurvaSawant11) [inline index.html]: You can remove this if not needed

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! You can merge the PR

File: scripts/header.js [hunk 2]
```diff
@@ -12,4 +14,5 @@ menu.addEventListener("click", () => {
     console.log("else case " + menuIcon.classList);
```
Comment (anik31) [inline scripts/header.js:12]: suggestion - you can remove console.log

File: styles/documentation/footer.css [hunk 3]
```diff
@@ -1,27 +1,33 @@
 .footer {
-  margin: 2rem 3rem;
+  /* margin: 2rem 3rem; */
```
Comment (anik31) [inline styles/documentation/footer.css]: suggestion - you can remove this if not required

**Review by anik31 — APPROVED** [review]
Nice work, the UI looks good in mobile view & hamburger is also working fine.

File: index.html (hunk 1 above)
Comment (vrx29) [reply index.html]: removed

File: scripts/header.js (hunk 2 above)
Comment (vrx29) [reply scripts/header.js:12]: done

File: styles/documentation/footer.css (hunk 3 above)
Comment (vrx29) [reply styles/documentation/footer.css]: removed

**Pull request merged**